action.yml

```diff
@@ -1,47 +1,47 @@
-name: 'ActionDebugger By Warpbuild'
-description: 'Debug your GitHub Actions Environment interactively by using SSH or a Web shell'
+name: "ActionDebugger By Warpbuild"
+description: "Debug your GitHub Actions Environment interactively by using SSH or a Web shell"
 branding:
   icon: terminal
   color: gray-dark
-author: 'Warpbuild'
+author: "Warpbuild"
 runs:
-  using: 'node20'
-  main: 'lib/index.js'
-  post: 'lib/index.js'
-  post-if: '!cancelled()'
+  using: "node20"
+  main: "lib/index.js"
+  post: "lib/index.js"
+  post-if: "!cancelled()"
 inputs:
   sudo:
-    description: 'If apt should be executed with sudo or without'
+    description: "If apt should be executed with sudo or without"
     required: false
-    default: 'auto'
+    default: "auto"
   install-dependencies:
-    description: 'Whether or not to install dependencies on linux (openssh-client, xz-utils)'
+    description: "Whether or not to install dependencies on linux (openssh-client, xz-utils)"
     required: false
```

Comment:
What is the use case for this option?

Reply:
If we are creating custom images, we can include these dependecies in the image itself, reducing the time to start tmate.

Reply:
Ah, cool.